README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository contains a plugin which provides the automatic Hazelcast member discovery using Eureka Service Registry.
 
-***NOTE:*** *hazelcast-eureka-one 2.0+* is compatible with *hazelcast 4.0+*, for older hazelcast versions you need to use *hazelcast-eureka-one 1.1.x*.
+***NOTE:*** *hazelcast-eureka-two* 3.x is compatible with *Eureka 2.x* and *Hazelcast 5.x*, for older hazelcast versions you need to use [eureka-one](/hazelcast/hazelcast-eureka/tree/eureka-one) branch.
```

Comment:
We have not break backward compatibility for Discovery SPI btw HZ 4.x and 5.x so hazelcast-eureka-two-3.x should be compatible with HZ 4.x, right?